### ISSUE TITLE
TEI 4.9.0

### ISSUE THREAD

**Issue by cmil** [dracor-org/dracor-schema#50]
~~[TEI 4.7.0](https://github.com/TEIC/TEI/releases/tag/P5_Release_4.7.0) has been released in November 2023.~~ [TEI 4.8.0](https://github.com/TEIC/TEI/releases/tag/P5_Release_4.8.0) was released on July 8, 2024.

Judging from the [current dracor.rng](https://github.com/dracor-org/dracor-schema/blob/01284fabc9aa45dfb712456592a3b3293ea9cf22/dracor.rng#L9) our ODD seems to be aligned with TEI 4.4.0. Should we get up to date with the recent Guidelines and what would that imply?

**Comment by lehkost** [comment]
[TEI 4.8.0](https://github.com/TEIC/TEI/releases/tag/P5_Release_4.8.0) was released on July 8, 2024 – we should align with this version.

**Comment by cmil** [comment]
[TEI 4.9.0](https://github.com/TEIC/TEI/releases/tag/P5_Release_4.9.0) was release in January.

In #94 (761b9f5d6b360167eaed182698fee05ce25b1d6d) we base our schema customisation on TEI 4.6.0 because that is the version that ships with oXygen 26. However the oXygen TEI framework can be upgraded independently, see https://github.com/TEIC/oxygen-tei/blob/master/oxygen-tei-plugin.md. So we may actually move on to the lates and greatest version.

On this occasion a friendly reminder goes to @ingoboerner and @lehkost that #94 is still open and waiting for review.

**Comment by lehkost** [comment]
I agree – if the only obstacle to upgrading to 4.9.0 is oXygen, we should proceed with the update, since, as you mentioned, it's possible to integrate the current TEI version into older versions of oXygen.
